ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1050,6 +1050,16 @@ port_channel_interfaces:
     esi: < EVPN Ethernet Segment Identifier (Type 1 format) >
     rt: < EVPN Route Target for ESI with format xx:xx:xx:xx:xx:xx >
     lacp_id: < LACP ID with format xxxx.xxxx.xxxx >
+    evpn_ethernet_segment:
+      redundancy: < Multihoming redundancy mode (all-active | single-active | default --> all-active) >
+      designated_forwarder_election:
+        algorithm: < modulus | preference | default --> modulus >
+        preference_value: < Preference value (0-65535) >
+        dont_preempt: < non-revertive DF election mode ( true | false | default --> false ) >
+        hold_time: < time seconds (1-1800) >
+        subsequent_hold_time: < time milliseconds (10-10000) >
+        candidate_reachability_required: < true | false | default --> false >
+        mpls_shared_index: < Label index value (1-1024) >
```

Comment:
This knob should be under ```evpn_ethernet_segment```. This is not specific to ```designated_forwarder_election```, I mean as per EOS command hierarchy.

![image](https://user-images.githubusercontent.com/88734745/149219352-2af518a5-4250-4a9d-8e2b-f5395940bf44.png)

Reply:
I would recommend the below data-model:

```
evpn_ethernet_segment:
  mpls:
    shared_index: < 1-1024 >
    tunnel_flood_filter_time: < 10-10000 >

Reply:
Sure, Tony. I think there was an error in the README where `mpls_shared_index` was placed under `designated_forwarder_election` as in the model its under `evpn_ethernet_segment`.

I'll update the model as you suggested.
```yaml
evpn_ethernet_segment:
  mpls:
    shared_index: < 1-1024 >
    tunnel_flood_filter_time: < 10-10000 >
```